arm9/src/unk_02031734.c

```diff
@@ -0,0 +1,2371 @@
+{
+    if (FUN_0202EE0C() != 0 && FUN_0202F950(0, 0, 0x200) != 0)
+    {
+        FUN_0202FA10();
+
+#ifdef NONMATCHING
+        UNK_021C5A00->unk0C = UNK_021C5A00->unk0C * UNK_021C5A00->unk14 + UNK_021C5A00->unk1C;
+        u32 r1 = ((s64)((u64)(UNK_021C5A00->unk0C) >> 32) * 0x20) >> 32;
+
+        FUN_02031D20(FUN_020324C8, r1);
+#else
+        asm(
+            // clang-format off
+            ldr r0, =UNK_021C5A00
+            ldr r4, [r0, #0x0]
+            add r5, r4, #0x0
+            add r5, #0xc
+            ldr r0, [r5, #0x8]
+            ldr r1, [r5, #0xc]
+            ldr r2, [r4, #0xc]
+            ldr r3, [r5, #0x4]
+            bl _ll_mul
+            ldr r2, [r5, #0x10]
+            ldr r3, [r5, #0x14]
+            add r0, r2, r0
+            adc r3, r1
+            str r0, [r4, #0xc]
+            mov r1, #0x0
+            ldr r0, =FUN_020324C8
+            lsr r2, r3, #0x1b
+            lsl r1, r1, #0x5
+            orr r1, r2
+            str r3, [r5, #0x4]
+            bl FUN_02031D20
+            // clang-format on
+        );
+#endif
+    }
+}
+
+THUMB_FUNC void FUN_02032628()
+{
+    if (FUN_0202E6F0() != 0)
+    {
+        FUN_02031D20(FUN_02032640, 0);
+    }
+}
+
+THUMB_FUNC void FUN_02032640()
+{
+    if (FUN_0202EE0C() != 0 && FUN_0202F950(0, 0, 0x200))
+    {
+        FUN_0202FA10();
+        FUN_02031D20(FUN_0203266C, 0x64);
+    }
+}
+
+THUMB_FUNC void FUN_0203266C()
+{
+    if (FUN_0202E8A0(UNK_021C5A00->unk3E) != 0 && FUN_0202FB18(UNK_021C5A00->unk3E) != 0)
+    {
+        FUN_02031D20(FUN_020326D0, 0x64);
+        return;
+    }
+
+    if (FUN_020311F0() != 0)
+    {
+        FUN_02031D20(FUN_020327B8, 0);
+        return;
+    }
+
+    if (UNK_021C5A00->unk3A != 0)
+    {
+        UNK_021C5A00->unk3A--;
+        return;
+    }
+
+    FUN_02031D20(FUN_020327B8, 0);
+}
+
+THUMB_FUNC void FUN_020326D0()
+{
+    if (FUN_020311F0() != 0)
+    {
+        FUN_02031D20(FUN_020327B8, 0);
+        return;
+    }
+
+    if (FUN_02030E7C(FUN_02031190()) != 0)
+    {
+        UNK_021C5A00->unk3D = 0;
+        FUN_020311D0(6, UNK_02105D5C);
+        FUN_02031D20(FUN_02032738, 0x78);
+        return;
+    }
+
+    if (UNK_021C5A00->unk3A != 0)
+    {
+        UNK_021C5A00->unk3A--;
+        return;
+    }
+
+    FUN_02031D20(FUN_020327B8, 0);
+}
+
+THUMB_FUNC void FUN_02032738()
+{
+    if (FUN_020311F0() != 0)
+    {
+        FUN_02031D20(FUN_020327B8, 0);
+        return;
+    }
+
+    if (UNK_021C5A00->unk3D == 2)
+    {
+        FUN_02031D20(FUN_020327B4, 0);
+        return;
+    }
+    else if (UNK_021C5A00->unk3D == 1)
+    {
+        FUN_0202DBE0();
+        FUN_02031D20(FUN_0203279C, 0);
+        return;
+    }
+
+    if (UNK_021C5A00->unk3A != 0)
+    {
+        UNK_021C5A00->unk3A--;
+        return;
+    }
+
+    FUN_02031D20(FUN_020327B8, 0);
+}
+
+THUMB_FUNC void FUN_0203279C()
+{
+    if (FUN_020311F0() != 0)
+    {
+        FUN_02031D20(FUN_020327B8, 0);
+    }
+}
+
+THUMB_FUNC void FUN_020327B4()
+{
```

Comment:
All these empty funcs are like this in assembly
```asm
	thumb_func_start FUN_020327B4
FUN_020327B4: ; 0x020327B4
	bx lr
	.balign 4
```

It could debugging yes